### ISSUE TITLE
TKO and multiple "after render"

### ISSUE THREAD

**Issue by hardhub** [knockout/tko#125]
Let's assume I have N different components which are parts of some general component C.  
Component C initially is hidden (let it be "if" binding but no matter), when it is loaded it starts loading of other components.  
During loading each child component fetches some data from server, displays it in forms / foreach loops etc.
After data is filled a component is displayed (for example _fade_ using CSS animation, opacity whatever).  
I want to continue some UI updates only when all children are absolutely ready (including DOM).  
How to handle the moment, when all N components fetched their data, filled it, and most important _rendered_ (animation completed), when I set _true_ for observable value of _if_ binding of C component.

In other words any chance to get something like this: `await(how?) commonVisibleObervable(true);`
